modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/GoClientCodegen.java

```diff
@@ -97,7 +97,7 @@ public GoClientCodegen() {
         typeMapping.put("boolean", "bool");
         typeMapping.put("string", "string");
         typeMapping.put("UUID", "string");
-        typeMapping.put("date", "time.Time");
+        typeMapping.put("date", "SwaggerDateType");
```

Comment:
Did it fail to compile with the template? time.Time used in this manner will cause a runtime error.

There may be a better way to handle it?